SwiftCompilerSources/Sources/Optimizer/PassManager/Context.swift

```diff
@@ -429,6 +429,33 @@ extension Type {
 //                          Builder initialization
 //===----------------------------------------------------------------------===//
 
+extension Instruction {
+  /// Returns self, unless self is a meta instruction, in which case the next
+  /// non-meta instruction is returned. Returns nil if there are no non-meta
+  /// instructions in the basic block.
+  fileprivate var skippingMetaInstructions: Instruction? {
```

Comment:
nitpick: a better name would be `nextNonMetaInstruction` ("skipping" is a verb and doesn't fit for a property, in contrast to a function)

Reply:
But I can't find a better name, so `nextNonMetaInstruction ` it is